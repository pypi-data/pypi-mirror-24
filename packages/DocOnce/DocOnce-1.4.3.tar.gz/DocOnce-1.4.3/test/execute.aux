\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Automatic execution of code blocks}{1}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}Plotting}{1}{section.1.1}}
\@writefile{toc}{\contentsline {section}{\numberline {1.2}Ignore output}{2}{section.1.2}}
\@writefile{toc}{\contentsline {section}{\numberline {1.3}Not executed}{2}{section.1.3}}
\@writefile{toc}{\contentsline {section}{\numberline {1.4}Code with errors}{3}{section.1.4}}
\@writefile{toc}{\contentsline {section}{\numberline {1.5}Opening files}{3}{section.1.5}}
\gdef\minted@oldcachelist{,
  default.pygstyle,
  default-pyg-prefix.pygstyle,
  F9B760D5F6119FC4B87BCCC153F3FCB4C6B333E1A7F7D5F955500D7AA7DA5925.pygtex,
  CDD071851FE1FE1312C14774C712731F40F1DDBD43D6C21A5C21CA8AB399B629.pygtex,
  4D45EC5F467E5D65FD2F63CBA1E5435A309E17391D9B9F7B537D970545DD384D.pygtex,
  6D7FCE9FEE471194AA8B5B6E47267F03309E17391D9B9F7B537D970545DD384D.pygtex,
  8E1455AFEE6C09A3A74A08B17273AD7A40F1DDBD43D6C21A5C21CA8AB399B629.pygtex,
  B445148A2A5252AC1169FC8701458F98309E17391D9B9F7B537D970545DD384D.pygtex,
  21D2920A4B613949464104D7AA9FB1F040F1DDBD43D6C21A5C21CA8AB399B629.pygtex,
  F3CB6641123C49A8B036B408A548BAAB309E17391D9B9F7B537D970545DD384D.pygtex,
  26AB0DB90D72E28AD0BA1E22EE510510309E17391D9B9F7B537D970545DD384D.pygtex,
  C211F234D30BB7C2ED8E685F7E16281B309E17391D9B9F7B537D970545DD384D.pygtex,
  9959AF584C9A80C08759A2ACCE75004A40F1DDBD43D6C21A5C21CA8AB399B629.pygtex,
  236BA5E77A5DE954A257893FAAAEDF34309E17391D9B9F7B537D970545DD384D.pygtex,
  D0FB2BACDDB5455AE49BE3EED05BBFFA40F1DDBD43D6C21A5C21CA8AB399B629.pygtex,
  0E897CA69071AB66CE7616D2945ADD26309E17391D9B9F7B537D970545DD384D.pygtex}
