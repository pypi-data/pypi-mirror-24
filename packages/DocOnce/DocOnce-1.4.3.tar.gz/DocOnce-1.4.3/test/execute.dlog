translating doconce text in execute.do.txt to pdflatex
translating doconce text in execute.do.txt to latex
output in execute.tex
translating doconce text in execute.do.txt to latex
output in execute.tex
translating doconce text in execute.do.txt to html
output in execute.html
translating doconce text in execute.do.txt to html
output in execute.html
translating doconce text in execute.do.txt to ipynb
output in execute.ipynb
translating doconce text in execute.do.txt to html
output in execute.html
translating doconce text in execute.do.txt to latex
translating doconce text in execute.do.txt to latex
translating doconce text in execute.do.txt to latex
output in execute.tex
translating doconce text in execute.do.txt to html
output in execute.html
translating doconce text in execute.do.txt to ipynb
output in execute.ipynb
translating doconce text in execute.do.txt to ipynb
translating doconce text in execute.do.txt to ipynb
output in execute.ipynb
translating doconce text in execute.do.txt to ipynb
translating doconce text in execute.do.txt to html
output in execute.html
translating doconce text in execute.do.txt to latex
output in execute.p.tex
