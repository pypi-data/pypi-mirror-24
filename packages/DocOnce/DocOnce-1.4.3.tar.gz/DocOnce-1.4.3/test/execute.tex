%%
%% Automatically generated file from DocOnce source
%% (https://github.com/hplgit/doconce/)
%%

% #define PREAMBLE

% #ifdef PREAMBLE
%-------------------- begin preamble ----------------------

\documentclass[%
oneside,                 % oneside: electronic viewing, twoside: printing
final,                   % draft: marks overfull hboxes, figures with paths
chapterprefix=true,      % "Chapter" word at beginning of each chapter
open=right,              % start new chapters on odd-numbered pages
10pt]{book}

\listfiles               %  print all files needed to compile this document

\usepackage{relsize,epsfig,makeidx,color,setspace,amsmath,amsfonts,amssymb}
\usepackage[table]{xcolor}
\usepackage{bm,ltablex,microtype}

\usepackage{graphicx}

% Packages for typesetting blocks of computer code
\usepackage{fancyvrb,framed,moreverb}

% Define colors
\definecolor{orange}{cmyk}{0,0.4,0.8,0.2}
\definecolor{tucorange}{rgb}{1.0,0.64,0}
\definecolor{darkorange}{rgb}{.71,0.21,0.01}
\definecolor{darkgreen}{rgb}{.12,.54,.11}
\definecolor{myteal}{rgb}{.26, .44, .56}
\definecolor{gray}{gray}{0.45}
\definecolor{mediumgray}{gray}{.8}
\definecolor{lightgray}{gray}{.95}
\definecolor{brown}{rgb}{0.54,0.27,0.07}
\definecolor{purple}{rgb}{0.5,0.0,0.5}
\definecolor{darkgray}{gray}{0.25}
\definecolor{darkblue}{rgb}{0,0.08,0.45}
\definecolor{darkblue2}{rgb}{0,0,0.8}
\definecolor{lightred}{rgb}{1.0,0.39,0.28}
\definecolor{lightgreen}{rgb}{0.48,0.99,0.0}
\definecolor{lightblue}{rgb}{0.53,0.81,0.92}
\definecolor{lightblue2}{rgb}{0.3,0.3,1.0}
\definecolor{lightpurple}{rgb}{0.87,0.63,0.87}
\definecolor{lightcyan}{rgb}{0.5,1.0,0.83}

\colorlet{comment_green}{green!50!black}
\colorlet{string_red}{red!60!black}
\colorlet{keyword_pink}{magenta!70!black}
\colorlet{indendifier_green}{green!70!white}

% Backgrounds for code
\definecolor{cbg_gray}{rgb}{.95, .95, .95}
\definecolor{bar_gray}{rgb}{.92, .92, .92}

\definecolor{cbg_yellowgray}{rgb}{.95, .95, .85}
\definecolor{bar_yellowgray}{rgb}{.95, .95, .65}

\colorlet{cbg_yellow2}{yellow!10}
\colorlet{bar_yellow2}{yellow!20}

\definecolor{cbg_yellow1}{rgb}{.98, .98, 0.8}
\definecolor{bar_yellow1}{rgb}{.98, .98, 0.4}

\definecolor{cbg_red1}{rgb}{1, 0.85, 0.85}
\definecolor{bar_red1}{rgb}{1, 0.75, 0.85}

\definecolor{cbg_blue1}{rgb}{0.87843, 0.95686, 1.0}
\definecolor{bar_blue1}{rgb}{0.7,     0.95686, 1}

%\setlength{\fboxsep}{-1.5mm}  % adjust cod_vpad/pro_vpad background box

%% Background for code blocks (parameter is color name)

%% pro/cod_vpad: gives some vertical padding before and after the text
%% (but has more simplistic code than _cod/pro_tight+cod/pro).
%% pro/cod_vpad can be used to enclose Verbatim or lst begin/end for code.
%% pro/cod calls _pro/cod_tight and has very little vertical padding,
%% used to enclose Verbatim and other begin/end for code.
%% (pro/cod is what the ptex2tex program could produce with the
%% Blue/BlueBar definitions in .ptex2tex.cfg.)

\newenvironment{cod_vpad}[1]{
   \def\FrameCommand{\colorbox{#1}}
   \MakeFramed{\FrameRestore}}
   {\endMakeFramed}

\newenvironment{_cod_tight}[1]{
   \def\FrameCommand{\colorbox{#1}}
   \FrameRule0.6pt\MakeFramed {\FrameRestore}\vskip3mm}
   {\vskip0mm\endMakeFramed}

\newenvironment{cod}[1]{
\bgroup\rmfamily
\fboxsep=0mm\relax
\begin{_cod_tight}{#1}
\list{}{\parsep=-2mm\parskip=0mm\topsep=0pt\leftmargin=2mm
\rightmargin=2\leftmargin\leftmargin=4pt\relax}
\item\relax}
{\endlist\end{_cod_tight}\egroup}

%% Background for complete program blocks (parameter 1 is color name
%% for background, parameter 2 is color for left bar)
\newenvironment{pro_vpad}[2]{
   \def\FrameCommand{\color{#2}\vrule width 1mm\normalcolor\colorbox{#1}}
   \MakeFramed{\FrameRestore}}
   {\endMakeFramed}

\newenvironment{_pro_tight}[2]{
   \def\FrameCommand{\color{#2}\vrule width 1mm\normalcolor\colorbox{#1}}
   \FrameRule0.6pt\MakeFramed {\advance\hsize-2mm\FrameRestore}\vskip3mm}
   {\vskip0mm\endMakeFramed}

\newenvironment{pro}[2]{
\bgroup\rmfamily
\fboxsep=0mm\relax
\begin{_pro_tight}{#1}{#2}
\list{}{\parsep=-2mm\parskip=0mm\topsep=0pt\leftmargin=2mm
\rightmargin=2\leftmargin\leftmargin=4pt\relax}
\item\relax}
{\endlist\end{_pro_tight}\egroup}

\usepackage{minted}
\usemintedstyle{default}

\usepackage[T1]{fontenc}
%\usepackage[latin1]{inputenc}
\usepackage{ucs}
\usepackage[utf8x]{inputenc}

\usepackage{lmodern}         % Latin Modern fonts derived from Computer Modern

% Hyperlinks in PDF:
\definecolor{linkcolor}{rgb}{0,0,0.4}
\usepackage{hyperref}
\hypersetup{
    breaklinks=true,
    colorlinks=true,
    linkcolor=linkcolor,
    urlcolor=linkcolor,
    citecolor=black,
    filecolor=black,
    %filecolor=blue,
    pdfmenubar=true,
    pdftoolbar=true,
    bookmarksdepth=3   % Uncomment (and tweak) for PDF bookmarks with more levels than the TOC
    }
%\hyperbaseurl{}   % hyperlinks are relative to this root

\setcounter{tocdepth}{3}  % levels in table of contents

% prevent orhpans and widows
\clubpenalty = 10000
\widowpenalty = 10000

% Redefine double page clear to make it a blank page without headers
% (from BYUTextbook)
\makeatletter
\def\cleardoublepage{\clearpage\if@twoside \ifodd\c@page\else
\hbox{}
\thispagestyle{empty}
\newpage
\if@twocolumn\hbox{}\newpage\fi\fi\fi}
\makeatother
% These commands fiddle with the space left for page numbers in the TOC
% (from BYUTextbook)
\makeatletter
%\renewcommand{\@pnumwidth}{2em}
%\renewcommand{\@tocrmarg}{2.85em}
\makeatother

% Make sure blank even-numbered pages before new chapters are
% totally blank with no header
\newcommand{\clearemptydoublepage}{\clearpage{\pagestyle{empty}\cleardoublepage}}
%\let\cleardoublepage\clearemptydoublepage % caused error in the toc

% --- end of standard preamble for documents ---


% insert custom LaTeX commands...

\raggedbottom
\makeindex
\usepackage[totoc]{idxlayout}   % for index in the toc
\usepackage[nottoc]{tocbibind}  % for references/bibliography in the toc

%-------------------- end preamble ----------------------

\begin{document}

% matching end for #ifdef PREAMBLE
% #endif

\newcommand{\exercisesection}[1]{\subsection*{#1}}

\input{newcommands_bfmath}
\input{newcommands_replace}

% ------------------- main content ----------------------

\chapter{Automatic execution of code blocks}

Compile this document with

\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
doconce format execute.do.txt --execute
\end{minted}
\end{cod}
\noindent

Hidden execution cells can be used to perform operations that need to be done
before the following cells are executed:









This is a cell that should execute automatically:

\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{python}
b = 2
c = a + b
print("The result is {}".format(c))
c
\end{minted}
\end{cod}
\noindent\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
The result is 3

\end{minted}
\end{cod}
\noindent
\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
3
\end{minted}
\end{cod}
\noindent


\section{Plotting}

This is a cell that should plot and output:

\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{python}
from pylab import *
x = linspace(0, 10, 100)
plot(x, x*x)
show()
\end{minted}
\end{cod}
\noindent\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-4-8e1455afee6c> in <module>()
----> 1 from pylab import *
      2 x = linspace(0, 10, 100)
      3 plot(x, x*x)
      4 show()

ImportError: No module named 'pylab'
\end{minted}
\end{cod}
\noindent


To improve quality when exporting to {\LaTeX}, the following code has automatically
been run to enable PDF export in notebooks.

\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
from IPython.display import set_matplotlib_formats
set_matplotlib_formats('png', 'pdf')
\end{minted}
\end{cod}
\noindent

\section{Ignore output}

Predefined output can be omitted by passing \Verb!--ignore_output! to DocOnce.
This will remove all environments ending with \texttt{out}.

\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{python}
a = 2
print(a)
\end{minted}
\end{cod}
\noindent\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
2

\end{minted}
\end{cod}
\noindent


\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
2
\end{minted}
\end{cod}
\noindent

\section{Not executed}

This is a code block that should not be executed:

\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
while True:
    i = 2
\end{minted}
\end{cod}
\noindent

This is some \emph{formatted} stuff, which is \textbf{underlined} and stuff and maybe even
contains $a + 2b$.

\section{Code with errors}

If code contains errors, it will still be run and the exception shown as part
of the output:

\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{python}
for a in range(10)
    print(a)
\end{minted}
\end{cod}
\noindent\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
  File "<ipython-input-6-9959af584c9a>", line 1
    for a in range(10)
                      ^
SyntaxError: invalid syntax

\end{minted}
\end{cod}
\noindent


\section{Opening files}

The working directory is the same as the .do.txt file.
You may want to use \texttt{os.chdir} to change the directory.

\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{python}
with open("../LICENSE") as f:
    print(f.read())
\end{minted}
\end{cod}
\noindent\begin{cod}{cbg_blue1}\begin{minted}[fontsize=\fontsize{9pt}{9pt},linenos=false,mathescape,baselinestretch=1.0,fontfamily=tt,xleftmargin=2mm]{text}
Copyright (c) 2007-2015, Hans Petter Langtangen <hpl@simula.no> and
Simula Resarch Laboratory.

All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are
met:

    * Redistributions of source code must retain the above copyright
      notice, this list of conditions and the following disclaimer.

    * Redistributions in binary form must reproduce the above copyright
      notice, this list of conditions and the following disclaimer in
      the documentation and/or other materials provided with the
      distribution.

    * Neither the name of Simula Research Laboratory nor the names of
      its contributors may be used to endorse or promote products
      derived from this software without specific prior written
      permission.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

Remarks:

The figure and movie files in doc/manual/* were made by the Doconce
author and is released under the same conditions as Doconce.



\end{minted}
\end{cod}
\noindent


% ------------------- end of main content ---------------

% #ifdef PREAMBLE
\end{document}
% #endif

