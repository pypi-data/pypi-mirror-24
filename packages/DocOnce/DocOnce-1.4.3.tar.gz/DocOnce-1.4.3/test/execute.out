\BOOKMARK [0][-]{chapter.1}{Automatic execution of code blocks}{}% 1
\BOOKMARK [1][-]{section.1.1}{Plotting}{chapter.1}% 2
\BOOKMARK [1][-]{section.1.2}{Ignore output}{chapter.1}% 3
\BOOKMARK [1][-]{section.1.3}{Not executed}{chapter.1}% 4
\BOOKMARK [1][-]{section.1.4}{Code with errors}{chapter.1}% 5
\BOOKMARK [1][-]{section.1.5}{Opening files}{chapter.1}% 6
